stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/config.py

```diff
@@ -2,7 +2,7 @@
 import os
 from typing import Set
 
-from elasticsearch import Elasticsearch
+from elasticsearch import AsyncElasticsearch, Elasticsearch
```

Comment:
we still need the sync Elasticsearch client b/c the bulk api does not support async yet.